packaging/windows/amazon-cloudwatch-agent-ctl.ps1

```diff
@@ -374,13 +374,11 @@ Function CWAConfig() {
         Remove-Item "${TOML}" -Force -ErrorAction SilentlyContinue
     } else {
         Write-Output "Start configuration validation..."
-        & cmd /c "`"$CWAProgramFiles\config-translator.exe`" --input `"${JSON}`" --input-dir `"${JSON_DIR}`" --output `"${TOML}`" --mode ${param_mode} --config `"${COMMON_CONIG}`" --multi-config ${multi_config} 2>&1"
-        CheckCMDResult  # Exit immediately if config-translator outputs any error
-
-        # Set ErrorActionPreference as Continue to continue on error when schema-test on toml file fails and
-        # return a UX-friendly message
+        & cmd /c "`"$CWAProgramFiles\config-translator.exe`" --input ${JSON} --input-dir ${JSON_DIR} --output ${TOML} --mode ${param_mode} --config ${COMMON_CONIG} --multi-config ${multi_config} 2>&1"
+        CheckCMDResult
+        # Let command pass so we can check return code and give user-friendly error-message
         $ErrorActionPreference = "Continue"
-        & cmd /c "`"${CWAProgramFiles}\amazon-cloudwatch-agent.exe`" --schematest --config `"${TOML}`" 2>&1" | Out-File $CVLogFile

```

Comment:
We not longer need the double quote for toml file path? I'm a bit ptsd on removing double quotes recently.

Reply:
@khanhntd can you explain? Reading through https://github.com/aws/amazon-cloudwatch-agent/pull/473 I can't tell if it's necessary or not

Reply:
It's only for consistency and also for future changing path. Since `amazon-cloudwatch-agent.exe` expect the TOML files as a static one:
* Windows: `ProgramData\\Amazon\\AmazonCloudWatchAgent\\amazon-cloudwatch-agent.toml`
* Linux/Mac: `/etc/opt/aws/amazon-cloudwatch-agent/etc/ common-config.toml`

So if we are changing `ProgramData` to `Program Files` or some path with escape path in the future, we will experience same issues